ucm2/sof-soundwire/sof-soundwire.conf

```diff
@@ -103,3 +109,51 @@ If.mics-array {
 		exec "-nhlt-dmic-info -o ${var:LibDir}/dmics-nhlt.json"
 	}
 }
+
+Include.hdmi-pcm.File "/common/pcm/hdmi.conf"
+
+If.Hdmi5-iec61937 {
```

Comment:
.. macro to reduce configuration text ..

Reply:
OK